community/kernel/src/main/java/org/neo4j/kernel/impl/store/MetaDataStore.java

```diff
@@ -808,7 +808,7 @@ public boolean closedTransactionIdIsOnParWithOpenedTransactionId()
         return onPar;
     }
 
-    public void logRecords( final Logger msgLog )
+    public void logRecords( final Consumer<String> consumer )
```

Comment:
It is actually `FormattedLogProvider.toOutputStream( System.out ).getLog( "something" ).infoLogger()` which is even uglier...